components/list/test/list.visual-diff.js

```diff
@@ -99,26 +108,28 @@ describe('d2l-list', () => {
 			{ name: 'focus', selector: '#button', action: () => focusButton('#button d2l-list-item-button') },
 			{ name: 'hover', selector: '#button', action: () => hover('#button d2l-list-item-button') }
 		] },
-		{ category: 'not selectable', tests: [
-			{ name: 'header', selector: '#noSelectableHeader' }
-		] },
 		{ category: 'selectable', tests: [
 			{ name: 'not selected', selector: '#selectable' },
 			{ name: 'not selected focus', selector: '#selectable', action: () => focusInput('#selectable [selectable]') },
 			{ name: 'not selected hover', selector: '#selectable', action: () => hover('#selectable [selectable]') },
-			{ name: 'not selected header', selector: '#selectableHeader' },
 			{ name: 'selected', selector: '#selectableSelected' },
 			{ name: 'selected focus', selector: '#selectableSelected', action: () => focusInput('#selectableSelected [selectable]') },
 			{ name: 'selected hover', selector: '#selectableSelected', action: () => hover('#selectableSelected [selectable]') },
-			{ name: 'some selected header', selector: '#selectableSomeSelectedHeader' },
-			{ name: 'all selected header', selector: '#selectableAllSelectedHeader' },
-			{ name: 'all selected header pages allowed', selector: '#selectableAllSelectedHeaderPages' },
 			{ name: 'slim', selector: '#selectableSlim' },
 			{ name: 'item-content', selector: '#selectableItemContent' },
 			{ name: 'item-content slim', selector: '#selectableItemContentSlim' },
 			{ name: 'skeleton', selector: '#selectableSkeleton' },
 			{ name: 'extended separators', selector: '#selectableSeparatorsExtended' }
 		] },
+		{ category: 'header', tests: [
```

Comment:
Just moved the header specific tests from above into this category, and added the two new tests for the sticky states.